db/sql/105_logs.sql

```diff
@@ -0,0 +1,32 @@
+CREATE SCHEMA logs;
+-- Important: tables here should be named as logs.<function name>_log
+
+-- Log levels
+-- Not modelled as enum to allow more convenient queries;
+-- enum values are handy in big data tables
+-- but logs.api_log is not such one.
+CREATE TABLE logs.log_level (
+  log_level        text          PRIMARY KEY
+);
+COMMENT ON TABLE logs.log_level IS
+'Log levels allowed for log tables log_level values.';
+INSERT INTO logs.log_level (log_level)
+VALUES ('info'), ('warning'), ('error'), ('debug');
+
+CREATE TABLE logs.importer_log (
+  id bigserial PRIMARY KEY,
+  timestamp timestamptz DEFAULT now(),
```

Comment:
This is not critical, but I'd rather use `log_timestamp` as a field name, like it was in the first commit introducing this schema I think. As a general practice, it's safer not to use Postgres keywords (`timestamp` is a data type) as identifiers, to avoid parsing and human errors. Sorry for missing this in my first review.

The same goes for `api_log` below.